docs/Makefile

```diff
@@ -14,6 +15,13 @@ help:
 
 .PHONY: help Makefile
 
+# Run the development server using sphinx-autobuild
+dev:
```

Comment:
What about `make html` (consistent with what we have on the docs elsewhere) or `make docs` instead of `make dev` (which is usually used for development and not docs)

Reply:
```
# Catch-all target: route all unknown targets to Sphinx using the new
# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
%: Makefile
	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

This snippet provided by Sphinx itself handles different modes like html, docs, etc

Reply:
Changed dev to docs

Reply:
the `make html` is already reserved by sphinx, so i don't think we should override this